dist/rpm/os-autoinst.spec

```diff
@@ -24,7 +24,8 @@ License:        GPL-2.0-or-later
 Group:          Development/Tools/Other
 Url:            https://github.com/os-autoinst/os-autoinst
 Source0:        %{name}-%{version}.tar.xz
-%define         build_requires autoconf automake gcc-c++ libtool opencv-devel > 3.0, pkg-config perl(Module::CPANfile) perl(Perl::Tidy) perl(Test::Compile) pkgconfig(fftw3) pkgconfig(libpng) pkgconfig(sndfile) pkgconfig(theoraenc) make
+#!BuildIgnore: opencv3-devel
```

Comment:
What is the purpose of this line?

Reply:
"spec: Force OBS to resolve choices on opencv-devel"

Reply:
I think this code which just looks like an out-commented line deserves a comment within the file. Nobody likes to dig in the Git history for this.

Reply:
done, updated